public/controllers/agent/components/register-agent.js

```diff
@@ -616,72 +914,402 @@ export const RegisterAgent = withErrorBoundary(
+                  <EuiCopy textToCopy={this.systemSelectorWazuhControl()}>
+                    {(copy) => (
+                      <div className="copy-overlay" onClick={copy}>
+                        <p><EuiIcon type="copy" /> Copy command</p>
+                      </div>
+                    )}
+                  </EuiCopy>
+                </div>
+                <EuiSpacer size='s' />
+                {textAndLinkToCheckConnectionDocumentation}
+              </EuiText>
+            </Fragment>
+          ),
+        },
+      ];
+
+      const buttonGroup = (legend, options, idSelected, onChange) => {
+        return (
+          <EuiButtonGroup
+            color="primary"
+            legend={legend}
+            options={options}
+            idSelected={idSelected}
+            onChange={onChange}
+            className={'osButtonsStyle'} />
+        )
+      }
+
+      const buttonGroupWithMessage = (legend, options, idSelected, onChange) => {
+        return (
+          <>
             <EuiButtonGroup
               color="primary"
-              legend="Choose the Operating system"
-              options={osButtons}
-              idSelected={this.state.selectedOS}
-              onChange={(os) => this.selectOS(os)}
+              legend={legend}
+              options={options}
+              idSelected={idSelected}
+              onChange={onChange}
+              className={'osButtonsStyle'}
             />
+            {this.state.selectedVersion == 'solaris10' || this.state.selectedVersion == 'solaris11' ? <EuiCallOut color="warning" className='message' iconType="iInCircle" title={
+              <span>
+                Might require some extra installation<EuiLink target="_blank" href="https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh-agent-package-solaris.html"> steps</EuiLink>.
+              </span>
+            }>
+            </EuiCallOut> : this.state.selectedVersion == '6.1 TL9' ? <EuiCallOut color="warning" className='message' iconType="iInCircle" title={
+              <span>
+                Might require some extra installation<EuiLink target="_blank" href="https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh-agent-package-aix.html"> steps</EuiLink>.
+              </span>
+            }>
+            </EuiCallOut> : this.state.selectedVersion == '11.31' ? <EuiCallOut color="warning" className='message' iconType="iInCircle" title={
+              <span>
+                Might require some extra installation<EuiLink target="_blank" href="https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh-agent-package-hpux.html"> steps</EuiLink>.
+              </span>
+            }>
+            </EuiCallOut> : <EuiCallOut color="warning" className='message' iconType="iInCircle" title={
+              <span>
+                The selected OS version reached its end of life (EOL). To install Wazuh follow our<EuiLink href="#"> guide</EuiLink>.
```

Comment:
**nitpick**: I am not sure if we do not have the link to the guide, but it is missing. The link only should affect the `guide` word.